test/performance.html

```diff
@@ -0,0 +1,83 @@
+<!-- open this file in your browser to test the performance of this package -->
```

Comment:
Do you want to explain the usage and/or purpose of this file either in main README.md or in a new README file in this sub-directory?

Reply:
I'm not sure how reliable this `.html` file is as a testing mechanism. That's why I didn't want to talk about it in the `readme.md`. The more reliable method is just using it in situ, which each user will have to try out for themselves.

Reply:
I might even want to just remove this file all together. Not sure.